service/templates/patches.html

```diff
@@ -28,22 +28,25 @@ <h1>
       Error loading data from server: [[patchesError]]
     </div>
 
-    <div  ng-if="patches.length > 0"
-          class="btn-group btn-group-sm pagination-fixed">
-      <span ng-if="currentPage > 0">
-        <a  class="btn btn-default"
-            style="cursor: pointer"
-            ng-click="previousPage()">
-          Newer
-        </a>
-      </span>
-      <span>
-        <a  class="btn btn-default"
-            style="cursor: pointer"
-            ng-click="nextPage()">
-          Older
-        </a>
-      </span>
+    <div class="form-inline" ng-if="patches.length > 0">
+      <div class="btn-group btn-group-sm header-pagination">
+        <input type="checkbox" id="queue" ng-model="filterCommitQueue" ng-change="toggleFilter()">
```

Comment:
is there a way to make this setting persist on subsequent page loads?

Reply:
Done!